index.html

```diff
@@ -1761,8 +1761,8 @@
         </h2>
         <p>
           An object that provides a <a>payment method</a> specific message used
-          by the merchant to process the transaction and determine successful
-          fund transfer. This data is returned by the <a>payment method</a>
+          by the merchant to process the transaction or validate the transaction
+          has been processed. This data is returned by the <a>payment method</a>
```

Comment:
Proposed:

            An object that provides information enabling the merchant to process or validate a transaction (depending on the <a>payment method</a>).